cluster/cluster_user.proto

```diff
@@ -68,12 +93,12 @@ message ClusterUser {
             string token = 1;
         }
     }
+}
 
-    message Delete {
-        message Req {
-            string username = 1;
-        }
-        message Res {}
+message User {
+    string username = 1;
+    oneof password_expiry_opt {
```

Comment:
if you check our other protocol definitions I believe you won't find that we label optional fields with `_opt` exception the `Options` ;) 

I would just end up with:
```
message User {
    string username = 1;
    oneof password_expiry {
        int64 password_expiry_days = 2;
    }
}
```

Note that you should be consistent with your casing! We should use `_` case everywhere

Reply:
Agreed, implemented.